src/huntsman/pocs/scheduler/observation/dithered.py

```diff
@@ -72,11 +57,23 @@ def field(self, values):
 
     @property
     def exposure_index(self):
-        exp_index = 0
+        _exp_index = 0
         with suppress(AttributeError):
-            exp_index = self.current_exp_num % len(self._exptime)
+            _exp_index = self.current_exp_num % len(self._exptime)
+
+        return _exp_index
+
+    def add_field(self, new_field, new_exptime):
```

Comment:
This is more like `add_position` as you are just adding a different point in the dither, correct?  I realize that's done via `Fields`, but I wonder how clear that will be.

Reply:
@wtgee The dithered observation is old code which I'm not intending to get into for this PR. I've just renamed the file.